docs/dwarf-op.rst

```diff
@@ -1,158 +1,180 @@
+    ``dwarf-op --list job_distribution`` to see how jobs are distributed.
+``--applyjobs "[" <job> [<job> ...] "]"``
+    Applies the listed jobs to the selected dwarves. Run
+    ``dwarf-op --list jobs`` to see available jobs.
+``--applyprofessions "[" <profession> [<profession> ...] "]"``
+    Applies the listed professions to the selected dwarves. Run
+    ``dwarf-op --list professions`` to see available professions.
+``--applytypes "[" <profession> [<profession> ...] "]"``
+    Applies the listed types to the selected dwarves. Run
+    ``dwarf-op --list dwf_types`` to see available types.
+``--renamejob <name>``
+    Renames the selected dwarves' custom professions to the specified name.
+
+.. _dorf_tables:
+
+Data tables
+-----------
+
+The data tables that ``dwarf-op`` uses are described below. They can be
+inspected with ``dwarf-op --list <table name>``.
+
+``job_distributions``
+    Defines thresholds for each column of distributions. The columns must add up
+    to the values in the thresholds row for that column.  Every other row
+    references an entry in the ``jobs`` table.
+
+``attrib_levels``
+    Defines stat distributions, used for both physical and mental attributes.
+    Each level gives a probability of a dwarf randomly being assigned an
+    attribute level, and it provides a mean and standard deviation for the
+    attribute's value.
+
+    The ``p`` values describe the normal distribution of stats (each ``p`` value
+    has a sub-distribution, which makes the bell curve not so bell-shaped).
+    Labours do not follow the same stat system and are more uniformly random,
+    which are compensated for in the description of jobs/professions.
+
+``jobs``
+    Defines ``dwarf-op``'s nameable jobs. Each job is comprised of required
+    professions, optional professions, probabilities for each optional
+    profession, a 'max' number of optional professions, and a list of types
+    (from the ``types`` table below) to apply to dwarves in the defined job.
+
+``professions``
+    These are a subset of the professions DF has. All professions listed will
+    match a profession dwarf fortress has built in, however not all the
+    built-ins are defined here.
+
+    Each profession is defined with a set of job skills which match the skills
+    built into Dwarf Fortress. Each skill is given a value which represents the
+    bonus a dwarf will get for this skill. The skills are added in a random
+    order, with the first few receiving the highest values (excluding the bonus
+    just mentioned). Thus the bonuses are to ensure a minimum threshold is
+    passed for certain skills deemed critical to a profession.
+
+``types``
+    These are a sort of archetype system for applying to dwarves. It primarily
+    includes physical attributes, but can include skills as well.
+
+    Each type has a probability of being applied to a dwarf just by pure luck -
+    this is in addition to types applied by other means. Each type also has a
```

Comment:
and in addition to the attrib levels. This can potentially bump attributes up higher, which maybe can be inferred but I'm not sure it can.

Reply:
reworded a bit to make this more clear.